bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-http.js

```diff
@@ -21,10 +22,14 @@ export default function (f7, isGraalJs) {
       const imageHeaderField = new Blockly.FieldImage(headerImage, 15, 15, undefined, this.onClickHeader)
       imageHeaderField.setTooltip('Add headers to the request.')
 
+      const imageQueryField = new Blockly.FieldImage(queryImage, 15, 15, undefined, this.onClickQuery)
+      imageQueryField.setTooltip('Add query to the request url.')
+
       this.appendValueInput('url')
         .setCheck('String')
         .appendField(imageTimeoutField, 'imgTimeout')
         .appendField(imageHeaderField, 'imgHeader')
+        .appendField(imageQueryField, 'imgHeader')
```

Comment:
I currently can‘t check how it looks like (only have my iPad with me), but is it correct that we have the `imgHeader` string here?

Reply:
No (even though it isn't really used to reference it later) - it should be imgQuery

Reply:
Indeed, should be changed, I didn’t notice in testing as the name reference is not used. I am not behind a PC, so difficult to do at the moment, but will do.

Reply:
@florian-h05 resolved now